### PR TITLE
clean up user code

### PR DESCRIPTION
**Pull request by abid-salahi**

### CODE REVIEW

**Comment by abid-salahi** [comment]
Not much changed here, just added a log to see whats going on on the front-end and also clean up useless params/code.

File: src/components/App.js [hunk 1]
```diff
@@ -65,7 +66,7 @@ const App = ({ dispatch }) => {
     } else {
       toRender(true);
     }
-  }, [dispatch]);
```
Comment (jenessatan) [inline src/components/App.js:68]: I think this was added to address console errors. @jacquelineyin

File: src/redux/actions/userActions.js [hunk 2]
```diff
@@ -152,7 +152,7 @@ export const loginUserAsync = (
           firstName: respData.user.firstName,
           lastName: respData.user.lastName,
           email: respData.user.email,
-          postalCode: "None",
+          postalCode: respData.user.postalCode,
```
Comment (jenessatan) [inline src/redux/actions/userActions.js:155]: are we adding postal code in somewhere?

**Review by jenessatan — APPROVED** [review]
just some clarification questions

File: src/redux/actions/userActions.js (hunk 2 above)
Comment (abid-salahi) [reply src/redux/actions/userActions.js:155]: yes during sign up

**Pull request merged**